Can't deploy newly-built images on cg-vm1 - Gunicorn error. again

```
Feb 05 14:44:05 cg-vm1.scilifelab.se systemd[1506]: scout@fix_rel_paths.service: Failed with result 'exit-code'.
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040192]: b22fdafbafd6911ed00f63fa83dcf6ad4d867d3b0cce67ca75becf4bc4dcb789
Feb 05 14:44:05 cg-vm1.scilifelab.se systemd[1506]: scout@fix_rel_paths.service: Main process exited, code=exited, status=1/FAILURE
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]: gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 525, in reap_workers
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     self.reap_workers()
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 242, in handle_chld
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     time.sleep(0.1)
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 393, in stop
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     self.stop()
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 342, in halt
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     self.halt(reason=inst.reason, exit_status=inst.exit_status)
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 229, in run
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     Arbiter(self).run()
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/app/base.py", line 72, in run
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     super().run()
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/app/base.py", line 231, in run
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 67, in run
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     sys.exit(run())
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/bin/gunicorn", line 8, in <module>
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]: Traceback (most recent call last):
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]: During handling of the above exception, another exception occurred:
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]: gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 525, in reap_workers
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     self.reap_workers()
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 242, in handle_chld
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     ready = select.select([self.PIPE[0]], [], [], 1.0)
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 357, in sleep
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:     self.sleep()
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 209, in run
Feb 05 14:44:05 cg-vm1.scilifelab.se podman[1040065]: Traceback (most recent call last):
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]: [2024-02-05 14:44:04 +0100] [6] [INFO] Worker exiting (pid: 6)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]: [2024-02-05 14:44:04 +0100] [5] [INFO] Worker exiting (pid: 5)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]: [2024-02-05 14:44:04 +0100] [7] [INFO] Worker exiting (pid: 7)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]: RecursionError: maximum recursion depth exceeded
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   [Previous line repeated 922 more times]
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return getattr(self.session, attr)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return getattr(self.session, attr)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return getattr(self.session, attr)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     self.make_declarative_base(Model)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 23, in __init__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     api = Alchy(Model=BASE)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 34, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from chanjo_report.server.extensions import api
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/views.py", line 7, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from .views import index_bp
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/__init__.py", line 2, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from .index import index_bp
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/__init__.py", line 2, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from .blueprints import report_bp, index_bp
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/config.py", line 2, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from .config import DefaultConfig
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/app.py", line 5, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from chanjo_report.server.app import configure_template_filters
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/home/worker/app/scout/server/app.py", line 39, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from .app import create_app
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/home/worker/app/scout/server/auto.py", line 4, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return _bootstrap._gcd_import(name[level:], package, level)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     mod = importlib.import_module(module)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return util.import_app(self.app_uri)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return self.load_wsgiapp()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     self.callable = self.load()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     self.wsgi = self.app.wsgi()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     self.load_wsgi()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     super().init_process()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/workers/gthread.py", line 92, in init_process
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     worker.init_process()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]: Traceback (most recent call last):
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]: [2024-02-05 14:44:04 +0100] [6] [ERROR] Exception in worker process
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]: [2024-02-05 14:44:04 +0100] [8] [INFO] Worker exiting (pid: 8)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]: RecursionError: maximum recursion depth exceeded
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   [Previous line repeated 922 more times]
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return getattr(self.session, attr)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return getattr(self.session, attr)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return getattr(self.session, attr)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     self.make_declarative_base(Model)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 23, in __init__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     api = Alchy(Model=BASE)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 34, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from chanjo_report.server.extensions import api
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/views.py", line 7, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from .views import index_bp
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/__init__.py", line 2, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from .index import index_bp
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/__init__.py", line 2, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from .blueprints import report_bp, index_bp
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/config.py", line 2, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from .config import DefaultConfig
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/app.py", line 5, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from chanjo_report.server.app import configure_template_filters
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/home/worker/app/scout/server/app.py", line 39, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from .app import create_app
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/home/worker/app/scout/server/auto.py", line 4, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return _bootstrap._gcd_import(name[level:], package, level)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     mod = importlib.import_module(module)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return util.import_app(self.app_uri)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return self.load_wsgiapp()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     self.callable = self.load()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     self.wsgi = self.app.wsgi()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     self.load_wsgi()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     super().init_process()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/workers/gthread.py", line 92, in init_process
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     worker.init_process()
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]: Traceback (most recent call last):
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]: [2024-02-05 14:44:04 +0100] [5] [ERROR] Exception in worker process
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]: RecursionError: maximum recursion depth exceeded
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   [Previous line repeated 922 more times]
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return getattr(self.session, attr)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return getattr(self.session, attr)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     return getattr(self.session, attr)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     self.make_declarative_base(Model)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 23, in __init__
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     api = Alchy(Model=BASE)
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 34, in <module>
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:     from chanjo_report.server.extensions import api
Feb 05 14:44:04 cg-vm1.scilifelab.se podman[1040065]:   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/views.py", line 7, in <module>
```

Oh look at the error I get when testing with docker-compose:

ModuleNotFoundError: No module named 'schug'

```
scout-scout-web-1  |   raw_paste_global_conf: []
scout-scout-web-1  |   strip_header_spaces: False
scout-scout-web-1  | [2024-02-05 14:41:19 +0000] [8] [INFO] Starting gunicorn 20.1.0
scout-scout-web-1  | [2024-02-05 14:41:19 +0000] [8] [DEBUG] Arbiter booted
scout-scout-web-1  | [2024-02-05 14:41:19 +0000] [8] [INFO] Listening at: http://0.0.0.0:8000 (8)
scout-scout-web-1  | [2024-02-05 14:41:19 +0000] [8] [INFO] Using worker: sync
scout-scout-web-1  | [2024-02-05 14:41:19 +0000] [14] [INFO] Booting worker with pid: 14
scout-scout-web-1  | [2024-02-05 14:41:19 +0000] [8] [DEBUG] 1 workers
scout-cli          | Traceback (most recent call last):
scout-cli          |   File "/venv/bin/scout", line 33, in <module>
scout-cli          |     sys.exit(load_entry_point('scout-browser', 'console_scripts', 'scout')())
scout-cli          |   File "/venv/bin/scout", line 25, in importlib_load_entry_point
scout-cli          |     return next(matches).load()
scout-cli          |   File "/usr/local/lib/python3.8/importlib/metadata.py", line 77, in load
scout-cli          |     module = import_module(match.group('module'))
scout-cli          |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
scout-cli          |     return _bootstrap._gcd_import(name[level:], package, level)
scout-cli          |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
scout-cli          |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
scout-cli          |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
scout-cli          |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
scout-cli          |   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
scout-cli          |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
scout-cli          |   File "/home/worker/app/scout/commands/__init__.py", line 1, in <module>
scout-cli          |     from .base import cli
scout-cli          |   File "/home/worker/app/scout/commands/base.py", line 14, in <module>
scout-cli          |     from scout.commands.download import download as download_command
scout-cli          |   File "/home/worker/app/scout/commands/download/__init__.py", line 1, in <module>
scout-cli          |     from .download_command import download
scout-cli          |   File "/home/worker/app/scout/commands/download/download_command.py", line 5, in <module>
scout-cli          |     from .ensembl import ensembl as ensembl_command
scout-cli          |   File "/home/worker/app/scout/commands/download/ensembl.py", line 8, in <module>
scout-cli          |     from scout.utils.ensembl_biomart_clients import EnsemblBiomartHandler
scout-cli          |   File "/home/worker/app/scout/utils/ensembl_biomart_clients.py", line 5, in <module>
scout-cli          |     from schug.load.biomart import EnsemblBiomartClient
scout-cli          | ModuleNotFoundError: No module named 'schug'
scout-cli exited with code 1
scout-scout-web-1  | [2024-02-05 14:41:25 +0000] [14] [ERROR] Exception in worker process
scout-scout-web-1  | Traceback (most recent call last):
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
scout-scout-web-1  |     worker.init_process()
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
scout-scout-web-1  |     self.load_wsgi()
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
scout-scout-web-1  |     self.wsgi = self.app.wsgi()
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
scout-scout-web-1  |     self.callable = self.load()
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
scout-scout-web-1  |     return self.load_wsgiapp()
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
scout-scout-web-1  |     return util.import_app(self.app_uri)
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
scout-scout-web-1  |     mod = importlib.import_module(module)
scout-scout-web-1  |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
scout-scout-web-1  |     return _bootstrap._gcd_import(name[level:], package, level)
scout-scout-web-1  |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
scout-scout-web-1  |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
scout-scout-web-1  |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
scout-scout-web-1  |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
scout-scout-web-1  |   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
scout-scout-web-1  |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
scout-scout-web-1  |   File "/home/worker/app/scout/server/auto.py", line 4, in <module>
scout-scout-web-1  |     from .app import create_app
scout-scout-web-1  |   File "/home/worker/app/scout/server/app.py", line 39, in <module>
scout-scout-web-1  |     from chanjo_report.server.app import configure_template_filters
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/app.py", line 5, in <module>
scout-scout-web-1  |     from .config import DefaultConfig
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/config.py", line 2, in <module>
scout-scout-web-1  |     from .blueprints import report_bp, index_bp
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/__init__.py", line 2, in <module>
scout-scout-web-1  |     from .index import index_bp
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/__init__.py", line 2, in <module>
scout-scout-web-1  |     from .views import index_bp
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/views.py", line 7, in <module>
scout-scout-web-1  |     from chanjo_report.server.extensions import api
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 34, in <module>
scout-scout-web-1  |     api = Alchy(Model=BASE)
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 23, in __init__
scout-scout-web-1  |     self.make_declarative_base(Model)
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
scout-scout-web-1  |     return getattr(self.session, attr)
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
scout-scout-web-1  |     return getattr(self.session, attr)
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
scout-scout-web-1  |     return getattr(self.session, attr)
scout-scout-web-1  |   [Previous line repeated 923 more times]
scout-scout-web-1  | RecursionError: maximum recursion depth exceeded
scout-scout-web-1  | [2024-02-05 14:41:25 +0000] [14] [INFO] Worker exiting (pid: 14)
scout-scout-web-1  | [2024-02-05 14:41:25 +0000] [8] [INFO] Shutting down: Master
scout-scout-web-1  | [2024-02-05 14:41:25 +0000] [8] [INFO] Reason: Worker failed to boot.
```

Reproduced locally using docker-compose (docker-compose up).

Seems to be chanjo-report related.

```
scout-scout-web-1  | [2024-02-05 16:07:23 +0000] [9] [INFO] Starting gunicorn 20.1.0
scout-scout-web-1  | [2024-02-05 16:07:23 +0000] [9] [DEBUG] Arbiter booted
scout-scout-web-1  | [2024-02-05 16:07:23 +0000] [9] [INFO] Listening at: http://0.0.0.0:8000 (9)
scout-scout-web-1  | [2024-02-05 16:07:23 +0000] [9] [INFO] Using worker: sync
scout-scout-web-1  | [2024-02-05 16:07:23 +0000] [15] [INFO] Booting worker with pid: 15
scout-scout-web-1  | [2024-02-05 16:07:24 +0000] [9] [DEBUG] 1 workers
scout-scout-web-1  | [2024-02-05 16:07:29 +0000] [15] [ERROR] Exception in worker process
scout-scout-web-1  | Traceback (most recent call last):
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
scout-scout-web-1  |     worker.init_process()
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
scout-scout-web-1  |     self.load_wsgi()
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
scout-scout-web-1  |     self.wsgi = self.app.wsgi()
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
scout-scout-web-1  |     self.callable = self.load()
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
scout-scout-web-1  |     return self.load_wsgiapp()
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
scout-scout-web-1  |     return util.import_app(self.app_uri)
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
scout-scout-web-1  |     mod = importlib.import_module(module)
scout-scout-web-1  |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
scout-scout-web-1  |     return _bootstrap._gcd_import(name[level:], package, level)
scout-scout-web-1  |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
scout-scout-web-1  |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
scout-scout-web-1  |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
scout-scout-web-1  |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
scout-scout-web-1  |   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
scout-scout-web-1  |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
scout-scout-web-1  |   File "/home/worker/app/scout/server/auto.py", line 4, in <module>
scout-scout-web-1  |     from .app import create_app
scout-scout-web-1  |   File "/home/worker/app/scout/server/app.py", line 39, in <module>
scout-scout-web-1  |     from chanjo_report.server.app import configure_template_filters
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/app.py", line 5, in <module>
scout-scout-web-1  |     from .config import DefaultConfig
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/config.py", line 2, in <module>
scout-scout-web-1  |     from .blueprints import report_bp, index_bp
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/__init__.py", line 2, in <module>
scout-scout-web-1  |     from .index import index_bp
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/__init__.py", line 2, in <module>
scout-scout-web-1  |     from .views import index_bp
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/views.py", line 7, in <module>
scout-scout-web-1  |     from chanjo_report.server.extensions import api
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 34, in <module>
scout-scout-web-1  |     api = Alchy(Model=BASE)
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 23, in __init__
scout-scout-web-1  |     self.make_declarative_base(Model)
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
scout-scout-web-1  |     return getattr(self.session, attr)
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
scout-scout-web-1  |     return getattr(self.session, attr)
scout-scout-web-1  |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
scout-scout-web-1  |     return getattr(self.session, attr)
scout-scout-web-1  |   [Previous line repeated 923 more times]
scout-scout-web-1  | RecursionError: maximum recursion depth exceeded
scout-scout-web-1  | [2024-02-05 16:07:29 +0000] [15] [INFO] Worker exiting (pid: 15)
scout-scout-web-1  | [2024-02-05 16:07:30 +0000] [9] [INFO] Shutting down: Master
scout-scout-web-1  | [2024-02-05 16:07:30 +0000] [9] [INFO] Reason: Worker failed to boot.
scout-scout-web-1 exited with code 3
scout-cli          | Traceback (most recent call last):
scout-cli          |   File "/venv/bin/scout", line 33, in <module>
scout-cli          |     sys.exit(load_entry_point('scout-browser', 'console_scripts', 'scout')())
scout-cli          |   File "/venv/bin/scout", line 25, in importlib_load_entry_point
scout-cli          |     return next(matches).load()
scout-cli          |   File "/usr/local/lib/python3.8/importlib/metadata.py", line 77, in load
scout-cli          |     module = import_module(match.group('module'))
scout-cli          |   File "/usr/local/lib/python3.8/importlib/__init__.py", line 127, in import_module
scout-cli          |     return _bootstrap._gcd_import(name[level:], package, level)
scout-cli          |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
scout-cli          |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
scout-cli          |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
scout-cli          |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
scout-cli          |   File "<frozen importlib._bootstrap_external>", line 843, in exec_module
scout-cli          |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
scout-cli          |   File "/home/worker/app/scout/commands/__init__.py", line 1, in <module>
scout-cli          |     from .base import cli
scout-cli          |   File "/home/worker/app/scout/commands/base.py", line 25, in <module>
scout-cli          |     from scout.server.app import create_app
scout-cli          |   File "/home/worker/app/scout/server/app.py", line 39, in <module>
scout-cli          |     from chanjo_report.server.app import configure_template_filters
scout-cli          |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/app.py", line 5, in <module>
scout-cli          |     from .config import DefaultConfig
scout-cli          |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/config.py", line 2, in <module>
scout-cli          |     from .blueprints import report_bp, index_bp
scout-cli          |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/__init__.py", line 2, in <module>
scout-cli          |     from .index import index_bp
scout-cli          |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/__init__.py", line 2, in <module>
scout-cli          |     from .views import index_bp
scout-cli          |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/blueprints/index/views.py", line 7, in <module>
scout-cli          |     from chanjo_report.server.extensions import api
scout-cli          |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 34, in <module>
scout-cli          |     api = Alchy(Model=BASE)
scout-cli          |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 23, in __init__
scout-cli          |     self.make_declarative_base(Model)
scout-cli          |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
scout-cli          |     return getattr(self.session, attr)
scout-cli          |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
scout-cli          |     return getattr(self.session, attr)
scout-cli          |   File "/venv/lib/python3.8/site-packages/chanjo_report/server/extensions.py", line 31, in __getattr__
scout-cli          |     return getattr(self.session, attr)
scout-cli          |   [Previous line repeated 927 more times]
scout-cli          | RecursionError: maximum recursion depth exceeded
scout-cli exited with code 1
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -15,7 +15,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - CLI scout update individual accepts subject_id
 - Update ClinVar inheritance models to reflect changes in ClinVar submission API
 - Handle variant-associated condition ID format in background when creating ClinVar submissions
-- Replace the code that downloads Ensembl genes, transcripts and exons with the Schug library
+- Replace the code that downloads Ensembl genes, transcripts and exons with the Schug web app
 ### Fixed
 - Text input of associated condition in ClinVar form now aligns to the left
 - Alignment of contents in the case report has been updated
@@ -23,7 +23,6 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Associate OMIM and/or ORPHA diagnoses with partial causatives
 - Visualization of partial causatives' diagnoses on case page: style and links
 - Revert style of pinned variants window on the case page
-
 
 ## [4.76]
 ### Added

--- a/requirements.txt
+++ b/requirements.txt
@@ -23,7 +23,6 @@ click
 cryptography
 defusedxml
 svglib
-schug
 cairosvg
 importlib_resources
 

--- a/scout/utils/ensembl_biomart_clients.py
+++ b/scout/utils/ensembl_biomart_clients.py
@@ -14,6 +14,7 @@ SCHUG_RESOURCE_URL: Dict[str, str] = {
     "exons": "/exons/ensembl_exons/?build=",
 }
 
+
 class EnsemblBiomartHandler:
     """A class that handles Ensembl genes, transcripts and exons downloads via schug-web."""
 

--- a/scout/utils/ensembl_biomart_clients.py
+++ b/scout/utils/ensembl_biomart_clients.py
@@ -1,35 +1,32 @@
 import logging
-from typing import Callable, Dict, Iterator
+from typing import Dict, Iterator
 
 import requests
-from schug.load.biomart import EnsemblBiomartClient
-from schug.load.ensembl import fetch_ensembl_exons, fetch_ensembl_genes, fetch_ensembl_transcripts
-from schug.models.common import Build as SchugBuild
 
 LOG = logging.getLogger(__name__)
+SCHUG_BASE = "https://schug-stage.scilifelab.se"
 
 BUILDS: Dict[str, str] = {"37": "GRCh37", "38": "GRCh38"}
 
-ENSEMBL_RESOURCE_CLIENT: Dict[str, Callable] = {
-    "genes": fetch_ensembl_genes,
-    "transcripts": fetch_ensembl_transcripts,
-    "exons": fetch_ensembl_exons,
+SCHUG_RESOURCE_URL: Dict[str, str] = {
+    "genes": "/genes/ensembl_genes/?build=",
+    "transcripts": "/transcripts/ensembl_transcripts/?build=",
+    "exons": "/exons/ensembl_exons/?build=",
 }
 
-
 class EnsemblBiomartHandler:
-    """A class that handles Ensembl genes, transcripts and exons downloads via schug."""
+    """A class that handles Ensembl genes, transcripts and exons downloads via schug-web."""
 
     def __init__(self, build: str = "37"):
         self.build: str = BUILDS[build]
 
-    def biomart_get(self, url: str) -> Iterator:
-        """Sends a request to Ensembl Biomart and returns the resource lines."""
+    def stream_get(self, url: str) -> Iterator:
+        """Sends a request to Schug web and returns the resource lines."""
         response: requests.models.responses = requests.get(url, stream=True)
         return response.iter_lines(decode_unicode=True)
 
     def stream_resource(self, interval_type: str) -> Iterator[str]:
-        """Fetches genes, transcripts or exons from a remote Ensembl biomart in the right genome build and saves them to file."""
+        """Use schug web to fetche genes, transcripts or exons from a remote Ensembl biomart in the right genome build and save them to file."""
 
         def yield_resource_lines(iterable) -> str:
             """Removes the last element from an iterator."""
@@ -39,11 +36,7 @@ class EnsemblBiomartHandler:
                 yield current
                 current = i
 
-        shug_client: EnsemblBiomartClient = ENSEMBL_RESOURCE_CLIENT[interval_type](
-            build=SchugBuild(self.build)
-        )
-
-        url: str = shug_client.build_url(xml=shug_client.xml)
+        shug_url: str = f"{SCHUG_BASE}{SCHUG_RESOURCE_URL[interval_type]}{self.build}"
 
         # return all lines except the last, which contains the "[success]" string
-        return yield_resource_lines(self.biomart_get(url))
+        return yield_resource_lines(self.stream_get(shug_url))

--- a/scout/utils/ensembl_biomart_clients.py
+++ b/scout/utils/ensembl_biomart_clients.py
@@ -4,7 +4,7 @@ from typing import Dict, Iterator
 import requests
 
 LOG = logging.getLogger(__name__)
-SCHUG_BASE = "https://schug-stage.scilifelab.se"
+SCHUG_BASE = "https://schug.scilifelab.se"
 
 BUILDS: Dict[str, str] = {"37": "GRCh37", "38": "GRCh38"}
 

--- a/scout/utils/ensembl_biomart_clients.py
+++ b/scout/utils/ensembl_biomart_clients.py
@@ -27,7 +27,7 @@ class EnsemblBiomartHandler:
         return response.iter_lines(decode_unicode=True)
 
     def stream_resource(self, interval_type: str) -> Iterator[str]:
-        """Use schug web to fetche genes, transcripts or exons from a remote Ensembl biomart in the right genome build and save them to file."""
+        """Use schug web to fetch genes, transcripts or exons from a remote Ensembl biomart in the right genome build and save them to file."""
 
         def yield_resource_lines(iterable) -> str:
             """Removes the last element from an iterator."""

--- a/tests/commands/download/test_download_ensembl_cmd.py
+++ b/tests/commands/download/test_download_ensembl_cmd.py
@@ -31,7 +31,7 @@ def test_print_ensembl_genes(mocker, genes37_handle):
 
     # GIVEN a patched call to schug
     # GIVEN a patched response from Ensembl Biomart, via schug
-    mocker.patch.object(EnsemblBiomartHandler, "biomart_get", return_value=genes37_handle)
+    mocker.patch.object(EnsemblBiomartHandler, "stream_get", return_value=genes37_handle)
 
     # GIVEN a temporary directory where the ensembl genes will be saved
     dir_name = tempfile.TemporaryDirectory()
@@ -51,7 +51,7 @@ def test_print_ensembl_transcripts(mocker, transcripts_handle):
     """Test print ensembl transcripts function."""
 
     # GIVEN a patched call to schug
-    mocker.patch.object(EnsemblBiomartHandler, "biomart_get", return_value=transcripts_handle)
+    mocker.patch.object(EnsemblBiomartHandler, "stream_get", return_value=transcripts_handle)
 
     # GIVEN a temporary directory where the ensembl genes will be saved
     dir_name = tempfile.TemporaryDirectory()
@@ -71,7 +71,7 @@ def test_print_ensembl_exons(mocker, exons_handle):
     """Test print ensembl exons function."""
 
     # GIVEN a patched call to schug
-    mocker.patch.object(EnsemblBiomartHandler, "biomart_get", return_value=exons_handle)
+    mocker.patch.object(EnsemblBiomartHandler, "stream_get", return_value=exons_handle)
 
     # GIVEN a temporary directory where the ensembl genes will be saved
     dir_name = tempfile.TemporaryDirectory()

--- a/tests/utils/test_ensembl_biomart_cients.py
+++ b/tests/utils/test_ensembl_biomart_cients.py
@@ -1,32 +1,31 @@
 from typing import Iterator
 
 import pytest
-from schug.demo import (
-    EXONS_37_FILE_PATH,
-    EXONS_38_FILE_PATH,
-    GENES_37_FILE_PATH,
-    GENES_38_FILE_PATH,
-    TRANSCRIPTS_37_FILE_PATH,
-    TRANSCRIPTS_38_FILE_PATH,
-)
-from schug.models.common import Build
 
+from scout.demo.resources import (
+    exons37_reduced_path,
+    exons38_reduced_path,
+    genes37_reduced_path,
+    genes38_reduced_path,
+    transcripts37_reduced_path,
+    transcripts38_reduced_path,
+)
 from scout.utils.ensembl_biomart_clients import EnsemblBiomartHandler
 from scout.utils.handle import get_file_handle
 
 SCHUG_BUILD_GENES_PATHS = [
-    (Build.build_37, GENES_37_FILE_PATH),
-    (Build.build_38, GENES_38_FILE_PATH),
+    ("37", genes37_reduced_path),
+    ("38", genes38_reduced_path),
 ]
 
 SCHUG_BUILD_TRANSCRIPTS_PATHS = [
-    (Build.build_37, TRANSCRIPTS_37_FILE_PATH),
-    (Build.build_38, TRANSCRIPTS_38_FILE_PATH),
+    ("37", transcripts37_reduced_path),
+    ("38", transcripts38_reduced_path),
 ]
 
 SCHUG_BUILD_EXONS_PATHS = [
-    (Build.build_37, EXONS_37_FILE_PATH),
-    (Build.build_38, EXONS_38_FILE_PATH),
+    ("37", exons37_reduced_path),
+    ("38", exons38_reduced_path),
 ]
 
 
@@ -39,7 +38,7 @@ def test_stream_resource_genes(build, path, mocker):
 
     # GIVEN a patched response from Ensembl Biomart, via schug
     mocker.patch.object(
-        EnsemblBiomartHandler, "biomart_get", return_value=get_file_handle(str(path))
+        EnsemblBiomartHandler, "stream_get", return_value=get_file_handle(str(path))
     )
 
     # THEN it should stream the gene resource when stream_resource in invoked
@@ -57,7 +56,7 @@ def test_stream_resource_transcripts(build, path, mocker):
 
     # GIVEN a patched response from Ensembl Biomart, via schug
     mocker.patch.object(
-        EnsemblBiomartHandler, "biomart_get", return_value=get_file_handle(str(path))
+        EnsemblBiomartHandler, "stream_get", return_value=get_file_handle(str(path))
     )
 
     # THEN it should stream the gene resource when stream_resource in invoked
@@ -75,7 +74,7 @@ def test_stream_resource_exons(build, path, mocker):
 
     # GIVEN a patched response from Ensembl Biomart, via schug
     mocker.patch.object(
-        EnsemblBiomartHandler, "biomart_get", return_value=get_file_handle(str(path))
+        EnsemblBiomartHandler, "stream_get", return_value=get_file_handle(str(path))
     )
 
     # THEN it should stream the gene resource when stream_resource in invoked

